testing/test_jsonnet.py

```diff
@@ -38,18 +38,32 @@
 from kubeflow.testing import test_util
 from kubeflow.testing import util
 
+# We should test all files which end in .jsonnet or .libsonnet
+# except ksonnet prototype definitions - they require additional
+# dependencies
+def should_test(f):
+  parts = os.path.splitext(f)
```

Comment:
_, ext = os.path.splitext(f)
Then use ext as parts[1] below.

I mean, shouldn't we test only libsonnet? So should_test will be return ext == '.libsonnet'